### PR TITLE
User docs tutorials - kpi-gauge chart

### PR DESCRIPTION
**Pull request by vulh-infragistics**

### CODE REVIEW

File: docs/user/tutorials-kpi-gauge.md
Comment (brianlagunas) [inline docs/user/tutorials-kpi-gauge.md:1]: We need to fix the table layout as the text is not aligned properly in the table cell

image

**Review by brianlagunas — CHANGES_REQUESTED** [review]

File: docs/user/tutorials-kpi-gauge.md
Comment (brianlagunas) [reply docs/user/tutorials-kpi-gauge.md:1]: would putting it in a two-column table look better? Title in the first column, image in the second column?

Comment (vulh-infragistics) [reply docs/user/tutorials-kpi-gauge.md:1]: I think you are viewing the pre-updated docs. I have updated the table to have a blank header and the links go to new lines.
I also updated to align the table cell content to be center:
image

Comment (hainv-ohio) [reply docs/user/tutorials-kpi-gauge.md:1]: Should be fixed now. @vulh-infragistics @brianlagunas

File: docs/user/tutorials-kpi-gauge.md [hunk 1]
```diff
@@ -75,15 +63,15 @@ can change this by modifying the "Type" field. In order to do so:
 |                                  |                                                                        |                                                                                                                                                |
```
Comment (hainv-ohio) [inline docs/user/tutorials-kpi-gauge.md:63]: @vulh-infragistics also fix the tables in here to remove the top empty headers

**Review by hainv-ohio — CHANGES_REQUESTED** [review]

File: docs/user/tutorials-kpi-gauge.md (hunk 1 above)
Comment (vulh-infragistics) [reply docs/user/tutorials-kpi-gauge.md:63]: Hi @hainv-ohio 
This should be fixed with the changes in the commit https://github.com/RevealBi/Documentation/pull/488/commits/4f73395437eaa01a88833a6e974919d9652adfc7

This removed the border inside the table. And it looks like:
image

**Pull request merged**